src/encoded/region_search.py

```diff
@@ -343,6 +349,11 @@ def region_search(context, request):
             chr=chromosome, start=start, end=end
         )
 
+    if expand_2kb:
+        start = int(start) - 2000
```

Comment:
I probably would have made this a parameter in case we want to change it some other number.